### PR TITLE
basic types exercise

### PR DESCRIPTION
**Pull request by arisbousios**

### CODE REVIEW

File: src/basic-types/exercise-solution.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+enum QuestionCategory {
+  GeneralKnowledge,
```
Comment (geogeorgiou) [inline src/basic-types/exercise-solution.ts]: please add assignment to each

`GeneralKnowledge = 'GeneralKnowledge'`

and so on

File: src/basic-types/exercise-solution.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  Science,
+  Math,
+  Programming,
+}
+
+type TrueFalseQuestion = {
```
Comment (geogeorgiou) [inline src/basic-types/exercise-solution.ts]: can we rename this as `Question` seems much more readable

File: src/basic-types/exercise-solution.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  category: QuestionCategory;
+  question: string;
+  correctAnswer: boolean;
+};
+
+const trueFalseQuestions: TrueFalseQuestion[] = [
```
Comment (geogeorgiou) [inline src/basic-types/exercise-solution.ts]: rename this to just `questions` makes more sense as well

File: src/basic-types/exercise.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+// Create a TypeScript program that models a simple quiz with true/false questions.
+// Use an array of questions objects "trueFalseQuestions" to store: category, question, correctAnswer
```
Comment (geogeorgiou) [inline src/basic-types/exercise.ts]: apply renamings in the exercise definition too :)

**Review by georgios-georgiou — APPROVED** [review]

**Pull request merged**